src/Orleans/Streams/Core/IStreamSubscriptionObserver.cs

```diff
@@ -7,29 +7,8 @@
 
 namespace Orleans.Streams.Core
 {
-    public interface IStreamSubscriptionObserver<T>
+    public interface IStreamSubscriptionObserver<T> 
     {
         Task OnNewSubscription(StreamSubscriptionHandle<T> handle);
```

Comment:
nit:  I know we've been bouncing around with naming, so feel free to ignore this, but now that the ideas have settled some, I think OnSubscribed here makes more sense.  It's basically telling the grain that they've been subscribed to the stream.  OnNewSubscription sound more like a notification of something that is happening, rather than something that has already happened.